docs/internals/release-process.txt

```diff
@@ -108,30 +138,29 @@ each version.
   regressions is much less of a concern.
 
 As a concrete example, consider a moment in time halfway between the release of
-Django 1.7 and 1.8. At this point in time:
+Django Y.0 and Y.1. At this point in time:
 
-* Features will be added to development master, to be released as Django 1.8.
+* Features will be added to development master, to be released as Django Y.1.
 
-* Critical bug fixes will be applied to the ``stable/1.7.x`` branch, and
-  released as 1.7.1, 1.7.2, etc.
+* Critical bug fixes will be applied to the ``stable/Y.0.x`` branch, and
```

Comment:
In general I like the switch from specific examples (1.7, 1.8) which go out of date to the generic `Y.0`, but it does make the sample branch name `stable/Y.0.x` more confusing, since it contains one placeholder 'Y' and one literal 'x'.

An alternative would be to just choose some far-future examples (e.g. 5.0 and 5.1).